multi-node/vagrant/Vagrantfile

```diff
@@ -8,7 +8,7 @@ require 'yaml'
 
 Vagrant.require_version ">= 1.6.0"
 
-$update_channel = "stable"
+$update_channel = "alpha"
```

Comment:
it's kind of odd that we're changing this unilaterally for all Vagrant clusters on behalf of rkt support, but honestly the vagrant stuff is strictly for dev use so alpha is probably preferable anyways